exercises/perfect-numbers/perfect_numbers_test.go

```diff
@@ -0,0 +1,48 @@
+package perfect
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestPositiveNumberRequiredErrorDefined(t *testing.T) {
+	if !reflect.TypeOf(ErrOnlyPositive).Implements(reflect.TypeOf((*error)(nil)).Elem()) {
```

Comment:
Hmm, if you think we don't need either, I am happy to trust your judgment. I am fine with its name starting with `Err` being indicative enough.

If we have a check anyway, I would prefer the compile time check. And that can just go as a single line in the other error test or on the top level.

So either changing or removing would be OK with me, and then after that I think it will be good to merge.